static/app/views/replays/detail/network/index.tsx

```diff
@@ -246,16 +313,64 @@ function NetworkList({replayRecord, networkSpans}: Props) {
           query={searchTerm}
         />
       </NetworkFilters>
-      <StyledPanelTable
-        columns={columns.length}
-        isEmpty={networkData.length === 0}
-        emptyMessage={t('No related network requests found.')}
-        headers={columns}
-        disablePadding
-        stickyHeaders
-      >
-        {networkData.map(renderTableRow)}
-      </StyledPanelTable>
+
+      <NetworkTable ref={networkTableRef}>
+        <AutoSizer>
+          {({width, height}) => (
+            <MultiGrid
+              ref={multiGridRef}
+              columnCount={columns.length}
+              columnWidth={({index}) => {
+                if (index === 0) {
+                  // For the status code column, we want to set a fixed width
+                  return Math.max(80, cache.columnWidth({index}));
+                }
+
+                // If the column is the path column, we want to take up the rest of the space
+                if (index === 1) {
+                  // We need to subtract the width of the other columns
+                  const otherColumnsWidth =
+                    columns.reduce((acc, _, i) => {
+                      if (i === 0) {
+                        return acc + Math.max(80, cache.columnWidth({index: i}));
+                      }
+
+                      if (i === 1) {
+                        return acc;
+                      }
+
+                      return acc + cache.columnWidth({index: i});
+                    }, 0) - scrollBarWidth;
```

Comment:
We need to subtract the scrollbar width so both grids (the header and the body) are in sync.

This is to avoid this kind of behavior: 

![image](https://user-images.githubusercontent.com/39612839/197640236-10841c9e-74c3-47b5-860f-e9400533bdd4.png)